v2/charts/azure-service-operator/values.yaml

```diff
@@ -92,6 +92,9 @@ image:
 # 'address' field defines the metrics binding address on which metrics
 metrics:
   enable: true
+  # secureMetrics controls whether metrics should be served via 'http' or 'https'.
+  # Flagging secureMetrics as true would use https
+  secureMetrics: false
```

Comment:
The comment in this yaml file reinforces my earlier question about whether secure metrics are turned on by default.